compliance_checker/cf/cf_1_6.py

```diff
@@ -2848,7 +2848,9 @@ def _cell_measures_core(self, ds, var, external_set, variable_template):
                         valid = False
                         reasoning.append(conversion_failure_msg)
                     else:
-                        if not cell_measure_units.is_convertible(Unit(f"m{exponent}")):
+                        if not cell_measure_units.is_convertible(
```

Comment:
Would be interested to measure the performance of this vs for example:

```
>>> cm_dimensionality = cell_measure_units.dimensionality()
>>> cm_symbolic_dim = {basis_unit._names.symbols[0]: order for basis_unit, order in cm_dimensionality.items()}
>>> cm_symbolic_dim == {'m': 2}
True
```

(note the private API usage for now - would like to expose symbol in a friendly form ASAP)

Reply:
Your suggestion is quite faster (again, this is poor's man benchmark:

```
%timeit cm_dimensionality = cell_measure_units.dimensionality(); cm_symbolic_dim = {basis_unit._names.symbols[0]: order for basis_unit, order in cm_dimensionality.items()}; cm_symbolic_dim == {'m': 2}
5.3 μs ± 148 ns per loop (mean ± std. dev. of 7 runs, 100,000 loops each)

%timeit cell_measure_units.is_convertible_to(ut_system.unit("m2"))
1.64 ms ± 24.6 μs per loop (mean ± std. dev. of 7 runs, 1,000 loops each)
```

Reply:
Thanks for measuring! I totally forgot that I actually made this easier:

```
>>> cell_measure_units.dimensionality() == {'meter': 2}
True
```

Performance shouldn't be any different.

Reply:
I'll investigate. Our tests are faulty though b/c everything is passing 😳